terraform/prod.tfvars

```diff
@@ -59,16 +59,17 @@ website_google_tag_id = "G-WCDTMFM6RG"
 cluster_container_insights_enabled = true
 
 // API / Backend
-api_enabled                           = true
-api_default_desired_task_count        = 3
-api_minumum_task_count                = 2
-api_maximum_task_count                = 5
-api_enable_grants_scraper             = false
-api_enable_grants_digest              = false
-api_enable_new_team_terminology       = true
-api_enable_my_profile                 = true
-api_enable_saved_search_grants_digest = true
-api_log_retention_in_days             = 30
+api_enabled                            = true
+api_default_desired_task_count         = 3
+api_minumum_task_count                 = 2
+api_maximum_task_count                 = 5
+api_enable_grants_scraper              = false
+api_enable_grants_digest               = false
+api_enable_new_team_terminology        = true
+api_enable_my_profile                  = true
+api_enable_saved_search_grants_digest  = true
+api_enable_grant_digest_scheduled_task = false
```

Comment:
As written, this PR uses this variable simultaneously to disable running the ECS scheduled task (`enabled = var.api_enabled && var.api_enable_grant_digest_scheduled_task`) and to short-circuit the script itself (`if (process.env.ENABLE_GRANT_DIGEST_SCHEDULED_TASK !== 'true') { return; }`). Does this feel redundant? Or helpful on the off chance the script gets triggered via some other means?

Reply:
Even if the configuration is ultimately sourced from one place, I think that being able to have a check at the infrastructure level as well as at the runtime level is a useful separation of concerns, so this feels right to me.